src/main/java/com/bite_sized_knowledge/bite_backend/domain/article/service/ArticleService.java

```diff
@@ -181,6 +184,39 @@ public ResArticlePage getArticles(Long blogId, String from, int limit, Member me
         return ResArticlePage.from(articles, next);
     }
 
+    @Transactional(readOnly = true)
+    public ResBookmarkPage getBookmarkedArticles(String fromArticleId, int limit, Member member) {
+        LocalDateTime fromUpdatedAt = null;
+        if (fromArticleId != null) {
+            Article article = entityManager.getReference(Article.class, fromArticleId);
+            ArticleBookmark cursorArticleBookmark = articleBookmarkRepository.findByArticle(article)
+                    .orElseThrow(() -> new BadRequestException("커서에 해당하는 글이 존재하지 않습니다."));
+            fromUpdatedAt = cursorArticleBookmark.getUpdatedAt();
+        }
+        log.info("fromUpdatedAt: {}", fromUpdatedAt);
+
+        List<ResArticle> articles = new ArrayList<>(
+                articleBookmarkRepository.findArticleBookmarksByMemberAndIsDeletedFalseOrderByUpdatedAtDescFromUpdatedAt(
+                        member.getId(),
+                        fromUpdatedAt,
+                        PageRequest.of(0, limit + 1)
+                )
+                .stream()
+                .map(ArticleBookmark::getArticle)
+                .map(article -> getResArticle(article, member))
+                .toList()
+        );
+
+        String next = null;
+        if (articles.size() > limit) {
+            // 다음 페이지가 있음 → 마지막 한 건은 제거하고 next 커서로 사용
+            ResArticle extraArticle = articles.removeLast();
+            next = extraArticle.id();
+        }
```

Comment:
1. 
```suggestion
        String next = null;
        if (articles.size() > limit) {
            // 다음 페이지가 있음 → 마지막 한 건은 제거하고 next 커서로 사용
            next = articles.removeLast().id();
        }
```
마지막 Article에 대한 객체를 굳이 변수로 정의할 필요 없이 이렇게 처리해도 좋을 것 같아요.

2. 
```suggestion
        String next = articles.size() > limit
                               ? articles.removeLast().id()
                               : null;
```
삼항연산자로 간략하게 해도 좋겠네요!